class7-week3/actions/clean.js

```diff
@@ -0,0 +1,18 @@
+const Todo = require('../models/todo')
+
+module.exports = function clean(request, response) {
+
+	//const id = request.params.id
+
+	Todo.clean( error => {
+	if (error) {
```

Comment:
Indentation - generally you want your callback to be nested one block deeper than the function call itself.